.idea/libraries/Dart_Packages.xml

```diff
@@ -5,174 +5,182 @@
         <entry key="async">
```

Comment:
Appreciate the PR, however, there are too many other files that are committed in it. Can just do the PR for the feature that you need?